qemu/tests/cfg/virtio_chardev_trace.cfg

```diff
@@ -0,0 +1,11 @@
+- virtio_chardev_trace:
+    only Linux
+    type = virtio_chardev_trace
+    start_vm = no
+    smp = 8
+    serials += ' agentctl '
+    serial_name_agentctl = 'agent-ctl-path'
+    chardev_backend_agentctl = 'pipe'
+    serial_filename_agentctl = '/tmp/virtio-trace/agent-ctl-path'
```

Comment:
please add depends on: https://github.com/avocado-framework/avocado-vt/pull/2430
And, base on the patch above,  update `serial_filename_**` to `chardev_path_**`, thanks!